packages/dx-react-chart-demos/src/demo-sources/export/bootstrap4/demo.jsx

```diff
@@ -0,0 +1,283 @@
+import React, { useState } from 'react';
+import {
+  ArgumentAxis,
+  ValueAxis,
+  Chart,
+  Legend,
+  BarSeries,
+  Title,
+} from '@devexpress/dx-react-chart-bootstrap4';
+import { Popover, PopoverBody } from 'reactstrap';
+import domtoimage from 'dom-to-image';
+import { Plugin, Template, TemplatePlaceholder } from '@devexpress/dx-react-core';
+import JsPDF from 'jspdf';
+import { scaleBand } from '@devexpress/dx-chart-core';
+import { ArgumentScale, Stack, Animation } from '@devexpress/dx-react-chart';
+
+import { gaming as data } from '../../../demo-data/data-vizualization';
+
+const options = [
+  'Print',
+  'PNG',
+  'JPEG',
+  'PDF',
+];
+
+const addKeyframe = (name, def) => {
+  if (typeof document === 'undefined') {
+    return;
+  }
+  const head = document.getElementsByTagName('head')[0];
+  let style = Array.from(head.getElementsByTagName('style'))
+    .find(node => node.dataset[Symbol('animation')]);
```

Comment:
How does it work? (each `Symbol('animation')` call returns new value)

Reply:
Done